.gitignore

```diff
@@ -34,6 +34,7 @@ sdks/python/LICENSE
 sdks/python/NOTICE
 sdks/python/README.md
 sdks/python/apache_beam/portability/api/*pb2*.*
+sdks/python/apache_beam/portability/common_urns.py
```

Comment:
<!--new_thread; commit:5f21d4ca23d7fee0a91196abf7a2359e2a8b805f; resolved:0-->
Is it necessary to have a generated file? Can't you just reflectively and lazily generate the module/class/constants? (whichever is easiest)